package.json

```diff
@@ -38,7 +39,7 @@
     "chalk": "2.4.2",
     "clipanion": "2.2.2",
     "datadog-metrics": "https://github.com/DataDog/node-datadog-metrics#12d16a80ea2a8846c91d80f5e127fb7b81b9f347",
-    "dd-trace": "0.27.0",
+    "dd-trace": "git+https://github.com/DataDog/dd-trace-js#a07b82c81fd894c1230e5cab75fa92e8f35c5ad6",
```

Comment:
for the moment it is not released so we have too settle with the commit that added the feature: https://github.com/DataDog/dd-trace-js/commit/a07b82c81fd894c1230e5cab75fa92e8f35c5ad6

Reply:
We should not link to branch commits, it's very brittle (PR can be squashed, branch rebased etc.) and this will break the CI and deploy here.

Reply:
I think even if you force push the commit is not deleted from github (e.g. [this commit](https://github.com/DataDog/datadog-ci/commit/3aa66a240e0448d761b666c4ce1bb35e7b0ba9c6) that I squashed). But yeah, we can hold off the merge until a proper release is done 👍

Reply:
using `0.29.2` now 😄